### PR TITLE
Release notes for java 7 deprecation and docker

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-536

Description of Changes:
* Release notes for the deprecation of Java 7 support
* Release notes for new dockerfiles
* Update Minimum Java version
* Fix a few other minor errors

DOD:
- [x] nav.yml has been updated if applicable
- [x] file has been included where required if applicable
- [x] files removed have been deleted, not just excluded from the build if applicable

Review
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes53.md [hunk 1]
```diff
@@ -29,6 +29,45 @@ The following new features and enhancements were added for the {{site.productnam
 
 The following premium plugins updates were released alongside {{site.productname}} 5.3.
 
+### Enhanced Media Embed
+
+The {{site.productname}} 5.3 release includes the following accompanying changes affecting **Self-hosted Essential, Professional, Premium, and Enterprise customers only**:
```
Comment (lnewson) [inline release-notes/release-notes53.md:34]: Should this say `Enterprise` here? As we really only have the other 3 now and the last was called "Pro" in the download page.

File: release-notes/release-notes53.md [hunk 2]
```diff
@@ -29,6 +29,45 @@ The following new features and enhancements were added for the {{site.productnam
...
+
+- The Java server-side component (`ephox-hyperlinking.war`) has been updated to version 2.3.0. This version requires Java 8 or higher. For information on the deprecation of Java 7 support, see: [Deprecated features](#deprecatedfeatures).
+- From {{site.productname}} 5.3, premium self-hosted bundles include the files required to deploy the server-side components as Docker containers. For information on Docker deployments of the server-side components, see: [Containerized service deployments]({{site.baseurl}}/enterprise/server/dockerservices/).
```
Comment (lnewson) [inline release-notes/release-notes53.md:37]: "build and deploy" maybe? as they need to build it first, before they can deploy.

File: release-notes/release-notes53.md [hunk 3]
```diff
@@ -41,9 +80,18 @@ For information on the PowerPaste plugin, see: [PowerPaste plugin]({{site.baseur
 
 The {{site.productname}} 5.3 release includes an accompanying release of the **Spellchecker Pro** premium plugin.
 
-**Spellchecker Pro** 2.0.2
+**Spellchecker Pro** 2.0.2
```
Comment (lnewson) [inline release-notes/release-notes53.md:83]: This should probably be `Spell Checker Pro`, as that's what it is elsewhere on the site.

File: release-notes/release-notes53.md (hunk 1 above)
Comment (tylerkelly13) [reply release-notes/release-notes53.md]: fixed by https://github.com/tinymce/tinymce-docs/pull/1482/commits/a371f8e8bf84dbb8784ef55e828d1af6ddb5b69f

File: release-notes/release-notes53.md (hunk 3 above)
Comment (tylerkelly13) [reply release-notes/release-notes53.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1482/commits/a371f8e8bf84dbb8784ef55e828d1af6ddb5b69f

File: release-notes/release-notes53.md (hunk 2 above)
Comment (tylerkelly13) [reply release-notes/release-notes53.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1482/commits/a371f8e8bf84dbb8784ef55e828d1af6ddb5b69f

**Pull request merged**